src/librustc/diagnostics.rs

```diff
@@ -329,6 +329,49 @@ loop. Without a loop to break out of or continue in, no sensible action can be
 taken.
 "##,
 
+E0271: r##"
```

Comment:
don't you also need to remove the other entry for `E0271` from the `register_diagnostics!` call below?

(I imagine this is why the Travis build failed.)

You can check this sort of thing quickly + locally by running `make tidy` on your end.